pom.xml

```diff
@@ -23,7 +23,10 @@
   <groupId>io.cdap.cdap</groupId>
   <artifactId>cdap</artifactId>
   <version>6.8.0-SNAPSHOT</version>
-  <packaging>pom</packaging>
+    <modules>
+        <module>cdap-error-api</module>
```

Comment:
why was this added here instead of where all the other modules are?

Reply:
No reason. I guess it was automatically added by the IDE. Moved it.